src/entities/user.ts

```diff
@@ -32,14 +32,17 @@ export class User {
     notificationsEnabled: boolean
 
     @Column({
-        type: 'text',
+        type: 'string',
         unique: true
     })
     email: string
 
-    @Column('text')
+    @Column('string')
     password: string
 
+    @Column('string')
+    token: string
```

Comment:
As an user can have only one token at the same time, he can no more be connected on mobile and on our web app at the same time, right ?